Assets/Scripts/Managers/NetworkManager.cs

```diff
@@ -75,4 +75,9 @@ public override void OnPlayerEnteredRoom(Photon.Realtime.Player newPlayer)
         base.OnPlayerEnteredRoom(newPlayer);
         Utility.Print("Another peep joined the room yo!", LogLevel.Error);
     }
+
+    public override void OnPlayerLeftRoom(Photon.Realtime.Player other)
+    {
+        Utility.Print("Another peep joined the room yo!", LogLevel.Error);
```

Comment:
Can we add info to this? e.g. see https://doc-api.photonengine.com/en/pun/v2/class_photon_1_1_realtime_1_1_player-members.html

Reply:
Also, this should say "Another peep *left* the room yo!"

Reply:
gone